README.md

```diff
@@ -0,0 +1,53 @@
+#DateBuilder#
+
+##Creating dates, doing simple operations or comparisons, transforming can be difficult in Java, even with latest versions##
```

Comment:
Incorrect markdown syntax: `##Creating dates, doing simple operations or comparisons, transforming can be difficult in Java, even with latest versions##` -> `## Creating dates, doing simple operations or comparisons, transforming can be difficult in Java, even with latest versions`

Reply:
fixed in https://github.com/LaRoueVerte/datebuilder/pull/1/commits/5c0622b4ebb5b9a952b21e0002ee024a104be21c as well